crates/worker/src/partition_processor_manager.rs

```diff
@@ -655,10 +655,11 @@ impl PartitionProcessorManager {
         match control_processor.command {
             ProcessorCommand::Stop => {
                 if let Some(processor_state) = self.processor_states.get_mut(&partition_id) {
+                    debug!(%partition_id, "Asked to stop partition processor by cluster controller");
```

Comment:
you know what would be cool? if you also added which cluster controller was the sender of this request.

Reply:
Yes, that would be useful. I'll check whether it can be easily added.